07-map/ref2.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import "fmt"
+
+func main() {
+	ref := map[int]int{}
+	modify(ref)
+	// START OMIT
+	var other = ref
+	other[1] *= 42
+	fmt.Println(other)
```

Comment:
I think it would make more sense to Print the map that isn't being modified in code to demonstrate that it still is modified.